### PR TITLE
Decimal compatibility

### PR DESCRIPTION
**Pull request by montaguegabe**
If on the main repo we change the line of impl.py that says:

```
"ANY_TO_JSON": lambda v: f"var{v} = json.dumps(var{v})\n",
```
to:
```
"ANY_TO_JSON": lambda v: f"var{v} = json.dumps(var{v}, cls=JSONEncoder)\n",
```

I imagine we need to make these changes to the local server. Feel free to close this if not necessary

### CODE REVIEW

**Review by Sand1929 — APPROVED** [review]

File: src/parsagon/server/server/tasks.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 from page_cache_server import get_cleaned_html
 from utils import build_structure, get_elem_data, recursive_list_subsample  # noqa
 
+from simplejson.encoder import JSONEncoder
```
Comment (Sand1929) [inline src/parsagon/server/server/tasks.py:12]: This import isn't actually necessary

Comment (Sand1929) [reply src/parsagon/server/server/tasks.py:12]: But having the package installed is necessary

Comment (montaguegabe) [reply src/parsagon/server/server/tasks.py:12]: How will generated code know what `JSONEncoder` refers to without the import?

Comment (Sand1929) [reply src/parsagon/server/server/tasks.py:12]: The import needs to be in the generated code; importing in this file doesn't do anything

**Pull request merged**